lms/urls.py

```diff
@@ -102,8 +102,10 @@
     url(r'^idea/', include('openedx.features.idea.urls')),
 
     # temporary experiment urls block start.
-    url(r'^organizations/', TemplateView.as_view(template_name='main_django.html')),
-    url(r'^learners/', TemplateView.as_view(template_name='main_django.html')),
+    url(r'^organizations/', TemplateView.as_view(template_name='main_django.html'),
```

Comment:
should not we remove these links since they are being served from unbounce and we won't be working on them either?

Reply:
Yes now they have created pages using unbounce but these 5 urls were added to show empty pages and then later on add content on those pages with convert, I am not sure if should remove them or not, should we take product team in  loop?